createdisk.sh

```diff
@@ -255,6 +255,10 @@ ${OC} --config $1/auth/kubeconfig create secret generic htpass-secret --from-lit
 ${OC} --config $1/auth/kubeconfig apply -f htpasswd_cr.yaml
 ${OC} --config $1/auth/kubeconfig create clusterrolebinding developer --clusterrole=sudoer --user=developer
 
+# Get cluster-kube-apiserver-operator image along with hash and tag it
+certImage=$(${OC} --config $1/auth/kubeconfig adm release info --image-for=cluster-kube-apiserver-operator)
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo podman tag $certImage openshift/cert-recovery
```

Comment:
When I test  this on a CRC image, the tagging goes well, however the tagging does not seem to pull the needed bits to ensure they are available locally? If I run 'pull' after 'tag', I get
```
Trying to pull quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:f955cd757f3a94daca69e3bc9d5f0659c270b411ce27478b8dfa479ca7b1c191...Getting image source signatures                                           
Copying blob 6c6726ab35bd skipped: already exists
Copying blob 1d2c4ce43b78 skipped: already exists
Copying blob 1c9f515fc6ab skipped: already exists
Copying blob c7de79b2356b skipped: already exists
Copying blob dcb1864555fe skipped: already exists
Copying config fff5ef4ac6 done
Writing manifest to image destination
Storing signatures
```
Does this mean it fetched the config from the net?

Reply:
@cfergeau when you get the image hash and search it with available images using `sudo podman images | grep <hash>` you can see that the image is already available. Can you directly try that image with tag, I am hoping that it doesn't have any bit extra but I have to make sure otherwise I will add `${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo podman pull $certImage` before it.